src/Classes/MyRadio/CoreUtils.php

```diff
@@ -749,4 +749,17 @@ public static function newPassword()
         'Vinyl',
         'Broadcasting',
     ];
+
+    /**
+     * Get the hash of the current git HEAD, used to append to CSS urls for caching
+     * @return mixed Either the hash or a boolean false
+     */
+    public static function getVersion() {
+        $branch_ref = trim(explode("ref: ", file_get_contents('../../.git/HEAD'))[1]);
+        if ( $hash = file_get_contents( sprintf( '../../.git/%s', $branch_ref ) ) ) {
```

Comment:
Single =
Also the spacing is weird

Reply:
Also, my god. This is awful in every way I can think of. Can you not?

Reply:
if this is only run once when myradio is initialised it's probably easier to just use the current date.